website_sale_ecommerce_description/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Sylvain Van Hoof <sylvain@okia.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Website Sale e-Commerce Description',
+    'category': 'Website',
+    'summary': 'Add a e-Commerce description',
+    'version': '11.0.1.0.1',
```

Comment:
Change version to 11.0.1.0.0
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers